cms/djangoapps/contentstore/views/course.py

```diff
@@ -536,41 +535,6 @@ def _get_rerun_link_for_item(course_key):
     return reverse_course_url('course_rerun_handler', course_key)
 
 
-def _deprecated_blocks_info(course_module, deprecated_block_types):
```

Comment:
I don't actually know what this is used for. I removed it because it is using the (now removed) course_structure api.

As far as I can tell, this is only used in course_overview.html were we show a warning that the course is using unsupported features. Is that something that is still needed?